src/Campaigns/Blocks/shared/components/CampaignCard/index.tsx

```diff
@@ -0,0 +1,73 @@
+import {__} from '@wordpress/i18n';
+import {getGoalDescription, getGoalFormattedValue} from '../../../CampaignGoal/utils';
+import {Campaign} from '@givewp/campaigns/admin/components/types';
+import {GiveCampaignOptions} from '@givewp/campaigns/types';
+
+declare const window: {
+    GiveCampaignOptions: GiveCampaignOptions;
+} & Window;
+
+import './styles.scss';
+
+export default ({showImage, showGoal, showDescription, campaign}: {
+    showImage: boolean,
+    showDescription: boolean,
+    showGoal: boolean,
+    campaign: Campaign
+}) => {
+
+    return (
+        <div
+            className="give-campaigns-component-campaign"
+            {...(campaign.pagePermalink && !window.GiveCampaignOptions.isAdmin && {
+                style: {
+                    cursor: 'pointer',
+                },
+                onClick: () => window.location = campaign.pagePermalink
```

Comment:
Are we sure this should be an onClick event and not an anchor tag wrapper?  Just trying to think what would be best semantically 🤔

Reply:
Yeah, wrapping everything into an anchor tag works, but most of the times that require a lot of `!important` flags to be added to CSS.

Reply:
Hmm, something to think about...  however I am running into an issue here where sometimes the onClick just reloads the page.  I'm thinking perhaps we need to additional conditionals for if the campaign page is enabled & published

Reply:
Yeah, that's because the campaign page is created when you hit the edit page button. For now, we can add a campaign page permalink check, but that doesn't resolve the issue. We should refactor this and create a campaign page when the campaign is created.

Reply:
Yeah I think we can improve that experience (not here), I still think we need to check if the landing page is enabled though or else the link will 404